connectors/migrations/20241211_fix_gdrive_parents.ts

```diff
@@ -60,19 +59,24 @@ async function migrate({
           id: {
             [Op.gt]: nextId,
           },
+          mimeType: "application/vnd.google-apps.folder",
         },
         order: [["id", "ASC"]],
         limit: QUERY_BATCH_SIZE,
       }
     );
 
-    for (const file of googleDriveFiles) {
+    loop: for (const file of googleDriveFiles) {
```

Comment:
nit: rename googleDriveFiles as googleDriveFolders since in that case we select for that

Reply:
I changed again and now gettng folders and csv , so wil keep "files" :-)